kratos/input_output/vtk_output.cpp

```diff
@@ -551,7 +585,7 @@ void VtkOutput::WriteVectorSolutionStepVariable(
     const TVarType& rVariable,
     std::ofstream& rFileStream) const
 {
-    KRATOS_DEBUG_ERROR_IF(rContainer.size() == 0) << "Empty container!" << std::endl;
+    KRATOS_WARNING_IF("VtkOutput" , rContainer.size() == 0) << "Empty container!" << std::endl;
 
     const int res_size = static_cast<int>((rContainer.begin()->FastGetSolutionStepValue(rVariable)).size());
```

Comment:
If you make it a warning it will crash here

Reply:
Can we add a skip?

Reply:
Skip is also not an option .. that is because when the file is opened with paraview it will crash paraview.